# Umeå Plant Science Centre Bioinformatics Facility

Welcome to the GitHub organisation from the Umeå Plant Science Centre Bioinformatics Facility (UPSCb in short).

## Repositories

### Active

|Repository|Description|DOI|
|---|---|---|
|[course-web](https://github.com/UPSCb/course-web)|A repository to generate a simple website in nodejs to serve workshop content|[![DOI](https://zenodo.org/badge/213847179.svg)](https://doi.org/10.5281/zenodo.15056554)|
|[Dockerfile](https://github.com/UPSCb/Dockerfile)|A collection of Dockerfile to create a variety of containers for bioinformatics analyses and workshops|[![DOI](https://zenodo.org/badge/71187353.svg)](https://doi.org/10.5281/zenodo.15056627)|
|[project-template](https://github.com/UPSCb/project-template)|A template to setup projects, based on the UPSCb guidelines for reproducibility|not relevant|
|[SeidRFile](https://github.com/UPSCb/SeidRFile)|A repository containing an R package to interact with file in the seidr format (sf) generated by [sei∂r](https://pubmed.ncbi.nlm.nih.gov/37313140/)|[![DOI](https://zenodo.org/badge/676073633.svg)](https://doi.org/10.5281/zenodo.15056546)|
|[UPSCb-common](https://github.com/UPSCb/UPSCb-common)|Common scripts and templates from the UPSCb. Cite it if you use it in your research.|[![DOI](https://zenodo.org/badge/206072841.svg)](https://doi.org/10.5281/zenodo.4001773)|

### Tutorials

If possible, please cite the tutorials if you (re-)use them for your teaching. Mind the license and get in touch if you use these commercially.

|Repository|Description|DOI|
|---|---|---|
|Tutorial_template|A template to create learnR tutorials|not-relevant|
|[Tutorial_01_data_science_intro](https://github.com/UPSCb/Tutorial_01_data_science_intro)||[![DOI](https://zenodo.org/badge/897459004.svg)](https://doi.org/10.5281/zenodo.15056619)|
|[Tutorial_02_exploratory_data_analysis](https://github.com/UPSCb/Tutorial_02_exploratory_data_analysis)||[![DOI](https://zenodo.org/badge/897748523.svg)](https://doi.org/10.5281/zenodo.15056548)|
|[Tutorial_03_differential_gene_expression](https://github.com/UPSCb/Tutorial_03_differential_gene_expression)||[![DOI](https://zenodo.org/badge/898483595.svg)](https://doi.org/10.5281/zenodo.15056617)|
|[Tutorial_04_gene_set_enrichment_analysis](https://github.com/UPSCb/Tutorial_04_gene_set_enrichment_analysis)||[![DOI](https://zenodo.org/badge/898663461.svg)](https://doi.org/10.5281/zenodo.15056615)|
|[Tutorial_05_ensemble_gene_set_enrichment_analysis](https://github.com/UPSCb/Tutorial_05_ensemble_gene_set_enrichment_analysis)||[![DOI](https://zenodo.org/badge/898690952.svg)](https://doi.org/10.5281/zenodo.15056556)|
|[Tutorial_06_differential_transcript_usage](https://github.com/UPSCb/Tutorial_06_differential_transcript_usage)||[![DOI](https://zenodo.org/badge/898679280.svg)](https://doi.org/10.5281/zenodo.15056623)|

### Published

|Repository|Description|Manuscript|DOI|
|---|---|---|---|
|[aba](https://github.com/UPSCb/aba)|Photoperiodic control of seasonal growth is mediated by ABA acting on cell-cell communication||[![DOI](https://zenodo.org/badge/425881953.svg)](https://doi.org/10.5281/zenodo.15056621)|
|[arabidopsis-floral-induction](https://github.com/UPSCb/arabidopsis-floral-induction)|Characterisation of metabolic changes associated with floral transition in Arabidopsis: variations in raffinose synthesis contribute to the determination of flowering time.||[![DOI](https://zenodo.org/badge/225628633.svg)](https://doi.org/10.5281/zenodo.15174456)|
|[algalAcclimatization](https://github.com/UPSCb/algalAcclimatization)|Algal acclimatization project|||
|[arabidopsis-aphids](https://github.com/UPSCb/arabidopsis-aphids)|PECTIN ACETYLESTERASE9 affects the transcriptome and metabolome and delays aphid feeding|||
|[aspen-tension-wood-development-gradient](https://github.com/UPSCb/aspen-tension-wood-development-gradient)|A high spatial resolution transcriptome analysis across tension wood developmental gradient in Populus tremula reveals key players in biosynthesis of contractile G-layer|TBA|TBA|
|[arabidopsis-nutrition-tor](https://github.com/UPSCb/arabidopsis-nutrition-tor)|Arabidopsis-nutrition-tor|||
|[arabidopsis-prn2-hub2](https://github.com/UPSCb/arabidopsis-prn2-hub2)|Arabidopsis prn2 hub2 mutants analysis|||
|[aspen-seasonal-senescence](https://github.com/UPSCb/aspen-seasonal-senescence)|aspen-seasonal-senescence|||
|[laccariaBicolorEcmDev](https://github.com/UPSCb/laccariaBicolorEcmDev)|The molecular timecourse of hybrid aspen/Laccaria bicolor ectomycorrhiza development|||
|[NorWood](https://github.com/UPSCb/NorWood)|||[![DOI](https://zenodo.org/badge/70584499.svg)](https://doi.org/10.5281/zenodo.15056625)|
|[pine-needles-fungi](https://github.com/UPSCb/pine-needles-fungi)|T4F pine needles pacbio amplicon sequencing|||
|[poplar-CRISPR-WGS](https://github.com/UPSCb/poplar-CRISPR-WGS)|The splicing genes SmEa and SmEb regulate plant development during vegetative growth in poplar|[DOI](https://doi.org/10.1186/s12870-025-07676-3)|[![DOI](https://zenodo.org/badge/706585581.svg)](https://doi.org/10.5281/zenodo.14892921)|
|[poplar-early-bud-break](https://github.com/UPSCb/poplar-early-bud-break)|Bud-break after winter dormancy is economically and environmentally important process in trees and shrubs from boreal and temperate latitudes but its molecular mechanisms are poorly understood. Previously we have reported the isolation and characterization of the EARLY BUD BREAK 1 (EBB1) gene, encoding an AP2/ERF transcription factor. We show here that EBB1 directly represses the poplar SVL (SHORT VEGETATIVE PHASE-LIKE), a MADS-box protein which was recently found to negatively regulate bud-break in Populus. We also report the identification and characterization of the EARLY BUD-BREAK 3 (EBB3) gene, which encodes another AP2/ERF transcription factor. We demonstrate that EBB3 is a positive regulator of bud-break and acts downstream of EBB1 and SVL. EBB3’s effect on bud-break is mediated via control of cell cycle by directly and positively regulating CYCLIND3.1 gene. Our results show the molecular architecture of a novel regulatory module that controls bud-break in poplar by controlling progression of cell cycle.|||
|[poplar-vegetative-growth-splicing](https://github.com/UPSCb/poplar-vegetative-growth-splicing)||||
|[poplarCambium](https://github.com/UPSCb/poplarCambium)||||
|[PoplarFT](https://github.com/UPSCb/PoplarFT)|FLOWERING LOCUS T Paralogs Control the Annual Growth Cycle in Populus Trees|[DOI](https://doi.org/10.1016/j.cub.2022.05.023)|[![DOI](https://zenodo.org/badge/480430992.svg)](https://doi.org/10.5281/zenodo.6448356)|
|[riboSeqPipeline](https://github.com/UPSCb/riboSeqPipeline)|RiboSeq pipeline|||
|[SeedTransNet-seidr](https://github.com/UPSCb/SeedTransNet-seidr)|Seidr analysis for SeedTransNet|||
|[seidrSpruce](https://github.com/UPSCb/seidrSpruce)|Running seidr on spruce compendia|||
|[spruce-cone-setting](https://github.com/UPSCb/spruce-cone-setting)|Here, we utilize the early cone-setting acrocona mutant to study reproductive phase change in the conifer Picea abies; a representative of the gymnosperm lineage. The acrocona mutant frequently initiates cone-like structures, called transition shoots, in positions where wild-type P. abies always produces vegetative shoots. By sequence analysis of mRNA and microRNA transcripts, we demonstrate that a member of the SQUAMOSA BINDING PROTEIN-LIKE (SPL) gene family, PaSPL1, is active in reproductive meristems, whereas two putative negative regulators of PaSPL1, microRNA156 and the conifer specific miR529, both are upregulated in vegetative meristems and transition shoots. By allele-specific assembly, we identify a mutation in the miRNA binding-site of the acrocona PaSPL1 allele. Data presented suggest that the mutation renders the acrocona allele micro-RNA resistant. Finally, by genotyping a segregating population of inbred acrocona trees, we show a clear co-segregation between the early cone-setting phenotype and trees homozygous for the acrocona mutation.|||
|[spruce-pine-light](https://github.com/UPSCb/spruce-pine-light)|Global gene expression analysis in etiolated and de-etiolated seedlings in conifers|||
|[spruce-seasonal-needles](https://github.com/UPSCb/spruce-seasonal-needles)|||[![DOI](https://zenodo.org/badge/206827829.svg)](https://doi.org/10.5281/zenodo.4949894)|
|[spruce-somatic-embryogenesis](https://github.com/UPSCb/spruce-somatic-embryogenesis)||||
|[spruce-wood-laser-microdissection](https://github.com/UPSCb/spruce-wood-laser-microdissection)|Blokhina et al., Plant Physiology, 2019|||
|[UPSCb](https://github.com/UPSCb/UPSCb)|The original repository of the UPSC Bioinformatics Facility, holds multiple published projects, their analyses and links to their datasets|TBA|[![DOI](https://zenodo.org/badge/74640960.svg)](https://doi.org/10.5281/zenodo.14002852)|
|[zygoticEmbryogenesis](https://github.com/UPSCb/zygoticEmbryogenesis)|Spruce zygotic embryogenesis|||

### Archive-able

Repository|Description|DOI|
|---|---|---|
|[RnaSeqTutorial](https://github.com/UPSCb/RnaSeqTutorial)|A long lived repository from an R package that was used for RNA-Seq workshops.|[![DOI](https://zenodo.org/badge/88738672.svg)](https://doi.org/10.5281/zenodo.15056820)|

### Forks

Repository|Description|
|---|---|
|[MARVEL](https://github.com/UPSCb/MARVEL)|A fork of the MARVEL repository used during the initial phase of the spruce v2.0 genome sequencing project|
